.github/workflows/test-git-tag.yaml

```diff
@@ -0,0 +1,30 @@
+name: test-git-tag
+
+on:
+  pull_request:
+    paths:
+      - "git-tag/**"
+      - ".github/workflows/test-git-tag.yaml"
+
+permissions: {}
+
+jobs:
+  test_git_tag_action:
+    runs-on: ubuntu-latest
+
+    permissions:
+      actions: read
+      contents: read
+
+    name: Test Git Tag
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          persist-credentials: false
+
+      - uses: ./git-tag
+        with:
+            git_tag_prefix: "v"
```

Comment:
the identention is a bit off, can you remove one tab/space?

Reply:
good catch, I pushed 4045dbd